frontend/packages/dev-console/src/components/monitoring/queries.ts

```diff
@@ -17,49 +17,17 @@ export interface MonitoringQuery {
 }
 
 export const metricsQuery = {
-  PODS_BY_CPU: 'CPU Usage',
-  PODS_BY_MEMORY: 'Memory Usage',
-  PODS_BY_FILESYSTEM: 'Filesystem Usage',
-  PODS_BY_NETWORK_IN: 'Receive Bandwidth',
-  PODS_BY_NETWORK_OUT: 'Transmit Bandwidth',
+  'CPU Usage': 'CPU Usage',
+  'Memory Usage': 'Memory Usage',
+  'Filesystem Usage': 'Filesystem Usage',
+  'Receive Bandwidth': 'Receive Bandwidth',
+  'Transmit Bandwidth': 'Transmit Bandwidth',
```

Comment:
In my opinion the Idea of having enum/const variables is that you can change the value (string) in one place and those variables used elsewhere should receive the changed string. But here in this case you need to change both Key  and Value, which is not very useful.